diff --git a/dist/client/theme-default/components/VPContent.vue b/dist/client/theme-default/components/VPContent.vue
index 3b2dbdf6e0fff8ec2cd36ca80c713b9bfe738733..66c835064907dd372a8afcfd249a916f003f7b43 100644
--- a/dist/client/theme-default/components/VPContent.vue
+++ b/dist/client/theme-default/components/VPContent.vue
@@ -16,7 +16,6 @@ const { hasSidebar } = useSidebar()
     id="VPContent"
     :class="{
       'has-sidebar': hasSidebar,
-      'is-home': frontmatter.layout === 'home'
     }"
   >
     <slot name="not-found" v-if="page.isNotFound"><NotFound /></slot>
@@ -69,15 +68,16 @@ const { hasSidebar } = useSidebar()
   width: 100%;
 }
 
-.VPContent.is-home {
-  width: 100%;
-  max-width: 100%;
-}
-
 .VPContent.has-sidebar {
   margin: 0;
 }
 
+@media (min-width: 768px) {
+  .VPContent {
+    width: 100vw;
+  }
+}
+
 @media (min-width: 960px) {
   .VPContent {
     padding-top: var(--vp-nav-height);
diff --git a/dist/client/theme-default/components/VPFooter.vue b/dist/client/theme-default/components/VPFooter.vue
index 3a21ff2b7f62a21a271cbdab6dd8da945b8d8321..e830ede950bd72aad5ffc815f7fc7015d2fb9cb9 100644
--- a/dist/client/theme-default/components/VPFooter.vue
+++ b/dist/client/theme-default/components/VPFooter.vue
@@ -40,6 +40,7 @@ const { hasSidebar } = useSidebar()
 
 @media (min-width: 768px) {
   .VPFooter {
+    width: 100vw;
     padding: 32px;
   }
 }
diff --git a/dist/client/theme-default/components/VPLocalNav.vue b/dist/client/theme-default/components/VPLocalNav.vue
index 15791dd5af695fac8134f4b31d6f560c3247ee77..83ce4af3f8f84b6ed9c801aaa6b1fb3d19caa96d 100644
--- a/dist/client/theme-default/components/VPLocalNav.vue
+++ b/dist/client/theme-default/components/VPLocalNav.vue
@@ -143,6 +143,10 @@ const classes = computed(() => {
 }
 
 @media (min-width: 768px) {
+  .VPLocalNav {
+    width: 100vw;
+  }
+
   .menu {
     padding: 0 32px;
   }
diff --git a/dist/client/theme-default/components/VPNav.vue b/dist/client/theme-default/components/VPNav.vue
index 577c8eceaded06dfe2e3a99f171b81a92a21615b..5c74e11cc706c671c50ca8e7fd42499309320374 100644
--- a/dist/client/theme-default/components/VPNav.vue
+++ b/dist/client/theme-default/components/VPNav.vue
@@ -49,6 +49,12 @@ watchEffect(() => {
   transition: background-color 0.5s;
 }
 
+@media (min-width: 768px) {
+  .VPNav {
+    width: 100vw;
+  }
+}
+
 @media (min-width: 960px) {
   .VPNav {
     position: fixed;
diff --git a/dist/client/theme-default/components/VPSidebar.vue b/dist/client/theme-default/components/VPSidebar.vue
index d6c7321930d9fcc1ab3bcbbffbd0a70f7eda3052..c53323364b9a4f984bec9a7a8df78769ecda3b00 100644
--- a/dist/client/theme-default/components/VPSidebar.vue
+++ b/dist/client/theme-default/components/VPSidebar.vue
@@ -100,8 +100,8 @@ watch(
 
 @media (min-width: 1440px) {
   .VPSidebar {
-    padding-left: max(32px, calc((100% - (var(--vp-layout-max-width) - 64px)) / 2));
-    width: calc((100% - (var(--vp-layout-max-width) - 64px)) / 2 + var(--vp-sidebar-width) - 32px);
+    padding-left: max(32px, calc((100vw - (var(--vp-layout-max-width) - 64px)) / 2));
+    width: calc((100vw - (var(--vp-layout-max-width) - 64px)) / 2 + var(--vp-sidebar-width) - 32px);
   }
 }
 
diff --git a/dist/client/theme-default/styles/base.css b/dist/client/theme-default/styles/base.css
index da61b764f4f471a8ed09594bcabedc86f7218922..0d5308d45ef3b99bd1bf8a6ec9dc4321df41d981 100644
--- a/dist/client/theme-default/styles/base.css
+++ b/dist/client/theme-default/styles/base.css
@@ -43,6 +43,7 @@ body {
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  overflow-x: hidden;
 }
 
 main {
