diff --git a/dist/client/app/router.js b/dist/client/app/router.js
index 251adb50d001cfe12b7d3ca2b963e3e54b1d78ec..d7275b355eb35f6fccec415ef72d49e64cec5993 100644
--- a/dist/client/app/router.js
+++ b/dist/client/app/router.js
@@ -28,7 +28,7 @@ export function createRouter(loadPageModule, fallbackComponent) {
                 href = url.pathname + url.search + url.hash;
             }
         }
-        if (inBrowser) {
+        if (inBrowser && href !== location.href) {
             // save scroll position before changing url
             history.replaceState({ scrollPosition: window.scrollY }, document.title);
             history.pushState(null, '', href);
diff --git a/dist/client/theme-default/components/VPContent.vue b/dist/client/theme-default/components/VPContent.vue
index c0bd52f6228b9a0f188aeeb126c58ee79d890672..37c217bf8dbf68199e69e06a34f3adc71aa5fe1d 100644
--- a/dist/client/theme-default/components/VPContent.vue
+++ b/dist/client/theme-default/components/VPContent.vue
@@ -20,7 +20,6 @@ const NotFound = inject('NotFound')
     id="VPContent"
     :class="{
       'has-sidebar': hasSidebar,
-      'is-home': frontmatter.layout === 'home'
     }"
   >
     <NotFound v-if="route.component === NotFound" />
@@ -58,15 +57,16 @@ const NotFound = inject('NotFound')
   width: 100%;
 }
 
-.VPContent.is-home {
-  width: 100%;
-  max-width: 100%;
-}
-
 .VPContent.has-sidebar {
   margin: 0;
 }
 
+@media (min-width: 768px) {
+  .VPContent {
+    width: 100vw;
+  }
+}
+
 @media (min-width: 960px) {
   .VPContent {
     padding-top: var(--vp-nav-height);
diff --git a/dist/client/theme-default/components/VPFooter.vue b/dist/client/theme-default/components/VPFooter.vue
index 49b26bcda5f991194986a8e8313f3c98ff1b1372..0de66c86b2294c5a48a4ae61eaa1962faeaf4044 100644
--- a/dist/client/theme-default/components/VPFooter.vue
+++ b/dist/client/theme-default/components/VPFooter.vue
@@ -30,6 +30,7 @@ const { hasSidebar } = useSidebar()
 
 @media (min-width: 768px) {
   .VPFooter {
+    width: 100vw;
     padding: 32px;
   }
 }
diff --git a/dist/client/theme-default/components/VPLocalNav.vue b/dist/client/theme-default/components/VPLocalNav.vue
index 1f829cc2dda3b4abee9a1b825b2e4213cd4de58b..a44a77a985526800affeda6d33685caf9e6ea852 100644
--- a/dist/client/theme-default/components/VPLocalNav.vue
+++ b/dist/client/theme-default/components/VPLocalNav.vue
@@ -79,6 +79,10 @@ function scrollToTop() {
 }
 
 @media (min-width: 768px) {
+  .VPLocalNav {
+    width: 100vw;
+  }
+
   .menu {
     padding: 0 32px;
   }
diff --git a/dist/client/theme-default/components/VPNav.vue b/dist/client/theme-default/components/VPNav.vue
index 1f7b554d092a8515a47edad7dd2093650cbbfcf5..2e9b6ce7beec84ac6c5d76beab1fb0e33bdac9e2 100644
--- a/dist/client/theme-default/components/VPNav.vue
+++ b/dist/client/theme-default/components/VPNav.vue
@@ -36,6 +36,12 @@ provide('close-screen', closeScreen)
   transition: background-color 0.5s;
 }
 
+@media (min-width: 768px) {
+  .VPNav {
+    width: 100vw;
+  }
+}
+
 @media (min-width: 960px) {
   .VPNav {
     position: fixed;
diff --git a/dist/client/theme-default/components/VPSidebar.vue b/dist/client/theme-default/components/VPSidebar.vue
index 2efd8c3d5184a3168fa46d7426d01b07087c9a16..7c64a00dab15b1af815b934f970e23f4a0e59c25 100644
--- a/dist/client/theme-default/components/VPSidebar.vue
+++ b/dist/client/theme-default/components/VPSidebar.vue
@@ -110,8 +110,8 @@ watchPostEffect(async () => {
 
 @media (min-width: 1440px) {
   .VPSidebar {
-    padding-left: max(32px, calc((100% - (var(--vp-layout-max-width) - 64px)) / 2));
-    width: calc((100% - (var(--vp-layout-max-width) - 64px)) / 2 + var(--vp-sidebar-width) - 32px);
+    padding-left: max(32px, calc((100vw - (var(--vp-layout-max-width) - 64px)) / 2));
+    width: calc((100vw - (var(--vp-layout-max-width) - 64px)) / 2 + var(--vp-sidebar-width) - 32px);
   }
 }
 
diff --git a/dist/client/theme-default/styles/base.css b/dist/client/theme-default/styles/base.css
index 247fb0ce0eac92ec83a907e821cb1d5a0d1c0615..998e0979ec2e6373d7e3095b12d50d22803d4f7a 100644
--- a/dist/client/theme-default/styles/base.css
+++ b/dist/client/theme-default/styles/base.css
@@ -30,6 +30,7 @@ body {
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  overflow-x: hidden;
 }
 
 main {
